src/frontend/http.rs

```diff
@@ -53,6 +53,7 @@ const BEARER_PREFIX: &str = "Bearer ";
 // We have a very lax CORS on this, so we don't mind browsers
 // caching it for as long as possible.
 const CORS_MAXAGE: u32 = 86400;
+const RUNTIME_HEADER: &str = "X-Seafowl-Runtime";
```

Comment:
Runtime sounds weird to me in this context, maybe something like:

```suggestion
const QUERY_TIME_HEADER: &str = "X-Seafowl-Query-Time";
```

Also, there's the matter of CORS-safelisted headers that @milesrichardson raised in the original issue. I'm fine with having this info be in a separate header (i.e. not in `Content-Type`), but we'd probably need to specify [`Access-Control-Expose-Headers`](https://developer.mozilla.org/en-US/docs/Glossary/CORS-safelisted_response_header#extending_the_safelist) as well then.